NugetLightProvider.cs

```diff
@@ -366,39 +366,36 @@ public void FindPackage(string name, string requiredVersion, string minimumVersi
 
             NormalizeVersion(request, ref requiredVersion, ref minimumVersion, ref maximumVersion);
 
-            try {
```

Comment:
Consider reverting the whitespace changes in this file as there are no code changes